docs/plugins/snacks/picker.md

```diff
@@ -0,0 +1,64 @@
+# Advanced Picker - File and Buffer Selection
+
+Fast, fuzzy-finding picker for files, buffers, and more with advanced filtering.
+
+## Features
+
+- **Fuzzy Search**: Intelligent fuzzy matching for quick file finding
+- **Multiple Sources**: Files, buffers, Git files, symbols, and more
+- **Preview Window**: Live preview of selected items
+- **Custom Actions**: Multiple actions for each picker type
+- **Fast Performance**: Optimized for large projects
+
+## Configuration
+
+```lua
+picker = { 
+  enabled = true,
+  layout = "vertical",    -- Layout style
+  preview = true,         -- Enable preview window
+  hidden = false,         -- Show hidden files
+}
+```
+
+## Usage
+
+### Commands
+- `:Pick files` - Find files in current directory
+- `:Pick buffers` - Switch between open buffers
+- `:Pick git_files` - Find files tracked by Git
+- `:Pick grep` - Search text in files
+- `:Pick symbols` - Find symbols in current file
```

Comment:
> - `:Pick files` - Find files in current directory
> - `:Pick buffers` - Switch between open buffers
> - `:Pick git_files` - Find files tracked by Git
> - `:Pick grep` - Search text in files
> - `:Pick symbols` - Find symbols in current file

These commands don't appear to exist, troubleshoot that issue.

Reply:
Additionally, could you please add some keybinds that use the leader key, possibly overwritting the fuzyfind that already exists with tree-sitter